.github/workflows/deploy.yml

```
@@ -0,0 +1,55 @@
+              type=sha
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Capture SHA tag
+        run: echo "SHA_TAG=${{ steps.meta.outputs.tags }}" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)